packages/services/src/module/operations.service.ts

```
@@ -0,0 +1,146 @@
+// types
+// import type { IModule, ILinkDetails, ModuleLink, TIssuesResponse } from "@plane/types";
+// services
+import { APIService } from "@/api.service";
+
+export class ModuleOperationService extends APIService {
+  constructor(baseURL: string) {
+    super(baseURL);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)